client/student/app/lecture/comment.form.html

```diff
@@ -1,6 +1,6 @@
 <form data-ng-submit="commentForm.submitComment()">
     <div class="input-group">
-      <input type="text" class="form-control" placeholder="Comment..." data-ng-model="commentForm.comment">
+      <input type="text" class="form-control" placeholder="Comment..." data-ng-model="commentForm.comment" maxlength="500">
```

Comment:
I think we should prefix our directives with "ifs". I.e. maxlength would be ifs-maxlength. This is to avoid future conflicts:

"Best Practice: In order to avoid collisions with some future standard, it's best to prefix your own directive names. For instance, if you created a <carousel> directive, it would be problematic if HTML7 introduced the same element. A two or three letter prefix (e.g. btfCarousel) works well. Similarly, do not prefix your own directives with ng or they might conflict with directives included in a future version of Angular."

https://docs.angularjs.org/guide/directive